README.md

```diff
@@ -73,4 +80,14 @@ A utility middleware with the following signature:
 (wrap-add-cache-headers [handler])
 ```
 
-This utility function returns a ring handler that will add `cache-control` headers ("private, max-age=0, no-cache") to `GET` and `PUT` if they are handled by the handler.
+This middleware returns a ring handler that will add `cache-control` headers ("private, max-age=0, no-cache") to `GET` and `PUT` requests if they are handled by the handler.
+
+## wrap-add-x-frame-options-deny
+
+A utility middleware with the following signature:
+
+```clj
+(wrap-add-x-frame-options-deny [handler])
+```
```

Comment:
These docs are a bit redundant - all middleware (middlewares?) are functions that take a ring handler and return a ring handler.